src/Contracts/FederationOptions.php

```diff
@@ -7,6 +7,7 @@
 class FederationOptions
 {
     private ?float $weight = null;
+    private ?string $remote = null;
```

Comment:
Is empty string allowed? If no, i'd typehint with non-empty-string

Reply:
I type-hinted the method to update the field. Is it what you had in mind?

Reply:
Yes, but the property also should declare that it allows only non empty string :)

Reply:
Added, too! Thanks